.github/workflows/verify.yaml

```diff
@@ -0,0 +1,21 @@
+name: Verify
+
+on:
+  pull_request:
+
+jobs:
+  prettier:
+    name: Prettier
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+        with:
+          # Make sure the actual branch is checked out when running on pull requests
+          ref: ${{ github.head_ref }}
+
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.3
```

Comment:
@saikumarrs I remember we had a pre-commit hook as well right?

Reply:
@chandumlg, it's better we configure this action to run for only the changed files.
Additionally, we can run a scheduled cleanup action to create baseline.

Reply:
Yeah. Either run only for changed filed or fix all and then run on all.

Reply:
Want to confirm one thing. This only checks if my source-code is not formatted along the lines that prettier has defined. Right ?!
What if there are lint failures & they somehow got committed ? They wouldn't be considered in this action right ?!

Reply:
Yeah, we need another action for it. However, we can only flag but not automatically fix them.

Reply:
Actually there is a way in which you can run only one command(eslint) with rules from eslint & prettier(ofcourse by disabling eslint-formatter rules) & flag if you find some changes. I tried it in one of our repositories. It is working well

Reply:
Does it also update the files with corrected formatting? If yes, great!

Reply:
Yes. It did

Reply:
Closing this pr then @sanpj2292 . Let's have eslint do the formatting and linting.